go/teams/transactions.go

```diff
@@ -21,28 +21,41 @@ import (
 // Not threadsafe.
 type AddMemberTx struct {
 	team     *Team
-	payloads []interface{} // *SCTeamInvites or *keybase1.TeamChangeReq
-
-	// We need two separate payloads for social invites and keybase
-	// invites because they cannot be mixed in one link.
-	isForSocialInvites map[int]bool
+	payloads []txPayload
 
 	completedInvites map[keybase1.TeamInviteID]bool
+	uidSeqno         map[keybase1.UID]int
```

Comment:
Add comment for what this is. If I understand it's the index of the first payload in which the uid is mentioned. Also not sure 'seqno' is right. Maybe `uidFirstPayload`